src/wui/playerdescrgroup.cc

```diff
@@ -76,17 +80,16 @@ PlayerDescriptionGroup::PlayerDescriptionGroup(UI::Panel* const parent,
 	d->btnPlayerTribe->sigclicked.connect(
 	   boost::bind(&PlayerDescriptionGroup::toggle_playertribe, boost::ref(*this)));
 	d->btnPlayerInit =
-	   new UI::Button(this, "player_initialization", xplayerinit, h / 2, w - xplayerinit, h / 2,
-	                  UI::ButtonStyle::kFsMenuSecondary, "", _("Initialization"));
-	d->btnPlayerInit->sigclicked.connect(
-	   boost::bind(&PlayerDescriptionGroup::toggle_playerinit, boost::ref(*this)));
+	   new UI::Dropdown<uint8_t>(this, "player_initialization", xplayerinit, h / 2, w - xplayerinit,
+	                             6, h / 2, _("Initialization"), UI::DropdownType::kTextual,
```

Comment:
```suggestion
	                             6, h / 2, "", UI::DropdownType::kTextual,
```
For consistency with the multiplayer game setup screen, and because the dropdown would become very wide with translations